iothub/device/tests/ModuleClientTests.cs

```diff
@@ -72,6 +72,22 @@ public void ModuleClient_CreateFromConnectionString_NoTransportSettings()
             Assert.IsNotNull(moduleClient);
         }
 
+        [TestMethod]
+        public void ModuleClient_CreateFromConnectionString_WithClientOptions()
+        {
+            // setup
+            var clientOptions = new ClientOptions
+            {
+                ModelId = "tempModuleId"
+            };
+
+            // act
+            var moduleClient = ModuleClient.CreateFromConnectionString(FakeConnectionString, clientOptions);
```

Comment:
If this fails, wouldn't it throw?

Reply:
I traced through the code, and yes. It seems like we throw if something fails. I am assuming you're suggesting the Assert is pointless because the moduleClient cannot be null, correct?